index.bs

```diff
@@ -75,10 +75,11 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
         text: abs; url: eqn-abs
         text: modulo; url: eqn-modulo
         url: sec-algorithm-conventions
-            text: !
-            text: ?
             text: ECMA-262 section 5.2
             text: conventions of the ECMAScript specification
+        url: sec-returnifabrupt-shorthands
+            text: !
+            text: ?
```

Comment:
Oh, being able to link directly to an explanation of what the shorthand does is amazing!